content/gherkin/reference.md

```diff
@@ -44,6 +44,7 @@ The primary keywords are:
 - `Background`
 - `Scenario Outline` (or `Scenario Template`)
 - `Examples`
+- `Rule`
```

Comment:
I'd prefer to have the `Rule` higher up in the list (see @TomerPacific 's PR)

Reply:
Corrected, I put it under `Feature` item.

NOTE: 
The old ordering represented the heading ordering on this page.
The reason I put the section w/ `Rule` further down is because it is a new addition and the other sections are more important (IMHO).